src/Navbar.js

```diff
@@ -0,0 +1,32 @@
+import React from "react"
+import PropTypes from "prop-types"
+import { Link, useMatch, useResolvedPath } from "react-router-dom"
+
+export default function NavBar() {
+  return (
+    <nav className='nav'>
+      <ul>
+        <CustomLink to='/'>Home</CustomLink>
+        <CustomLink to='/team'>Team</CustomLink>
```

Comment:
linking to an undefined route with `/team`. If we actually click on this link it routes us to the 404 error view. This is because the team route is defined as `./team`. Be sure to locally test and make sure everything works! Refer to comment above for requested changes.